samples/nvidia_car_classification/module.yml

```diff
@@ -7,6 +7,32 @@ parameters:
     module: samples.nvidia_car_classification.overlay
     # specify the drawfunc's python class from the module
     class_name: Overlay
+    rendered_objects:
+      Primary_Detector:
+        Car:
+          bbox:
+            border_color: 00FF00FF  # Green
+            background_color: '00000000'  # transparent
+            thickness: 2
+          label:
+            font_color: FFFFFFFF  # White
```

Comment:
I suggest using a single notation style. Since zeros without quotation are interpreted as numbers, use quotation everywhere. In other configuration files also use the color in quotes.

Reply:
Added quotation marks.